demosplan/DemosPlanCoreBundle/Controller/Statement/DemosPlanStatementTagController.php

```diff
@@ -180,8 +181,8 @@ public function tagListEditAction(
                     $result = $statementHandler->createTagFromTopicId($requestPost['r_createtags'], $newTag, $procedure);
                     $this->getMessageBag()->add('confirm', 'confirm.tag.created');
                     $anchor = $result->getId();
-                } catch (Exception) {
-                    $this->getMessageBag()->add('warning', 'warning.tag.created');
+                } catch (DuplicatedTagTitleException $e) {
```

Comment:
What should happen when another exception like `BadRequestException` is thrown? It is not caught any more. Is this correct?